app/build.gradle

```diff
@@ -1,37 +1,21 @@
 apply plugin: 'com.android.application'
-import com.android.builder.core.DefaultManifestParser
 
 android {
-    signingConfigs {
```

Comment:
Maybe I should put this back, but let me know if so. I wasn't sure about what to do with the things in this file.

Reply:
Yeah, I need this to be able to put it on the Play Store...  perhaps it needs separate debug and release build configs or something...

Reply:
Oh, yeah, I have a release config already, just needs an additional config in the buildTypes block that doesn't have that I think.

Reply:
ok, the signing config I can do all in local configuration, yay!  I will move that there and this can go as-is. :-)